### ISSUE TITLE
Stylus button shorcut: Add drag Mode

### ISSUE THREAD

**Issue by Leon0402** [flxzt/rnote#133]
Would be great if I can set up a shortcut so I can drag the paper. This is an essential tool when working efficiently in my opinion. 

Presumably this should be added as a general tool. In xournal it's the "hand" mode.

**Comment by flxzt** [comment]
definitely. Having it as a tool and having shortcuts actions for the tools is on my todo

**Comment by giannissc** [comment]
You can actually do that now by using the non-canvas area (gray area) to drag the paper

**Comment by flxzt** [comment]
it also is now implemented as a tool. You can also set it up as a shortcut with switching the tool page. The selected tool is remembered.

**Comment by Leon0402** [comment]
Cool! I will give it a try once it is released and let you know how it works
